grid-demo/src/app/angular/final-demo/final-demo.component.scss

```diff
@@ -2,6 +2,17 @@ clr-datagrid {
   height: 500px;
 }
 
+clr-dg-row {
+  height: 72px;
+}
```

Comment:
The active cell border isn't the correct size. I think it would be better to remove this and set `height: 50px;` (72px - 11px vertical padding) in the `.cell-wrapper` rule.

Reply:
I already have this change locally from testing. I can push it and approve if you want.

Reply:
yes, please do

Reply:
yes, please do. I may have to adjust the line-clamping number

Reply:
Okay, pushed.